### PR TITLE
Deleted Themed.tsx, updated style of Sign-up screen.

### LINKED ISSUE #7: Remove default Themed.tsx component
It was used in the default expo project, so now it is an orphan and we should remove it from the codebase.

### PR DESCRIPTION
**Pull request by mihaildanilov**

### CODE REVIEW

File: screens/SignUpScreen.tsx [hunk 1]
```diff
@@ -1,22 +1,22 @@
-import { StyleSheet } from 'react-native';
-import { View } from '../components/Themed';
+import React from 'react';
```
Comment (edwinlzrvch) [inline screens/SignUpScreen.tsx]: do you need to import React? It should work without it

Comment (kai-353) [reply screens/SignUpScreen.tsx]: I think you need React if you're going to use JSX. Not sure about that it .tsx files tho but that applies to .js files

Comment (edwinlzrvch) [reply screens/SignUpScreen.tsx]: App.tsx doesn't have React import + you can try to delete it any .tsx file and it will work

Comment (mihaildanilov) [reply screens/SignUpScreen.tsx]: If you both agree, then i just delete it, if you don't see any reason in it.

Comment (edwinlzrvch) [reply screens/SignUpScreen.tsx]: Quick updated: we have 17.0.1 React version so import React from 'react' is [no more needed](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html)

Comment (mihaildanilov) [reply screens/SignUpScreen.tsx]: Ok, good to know

**Review by edwinlzrvch — APPROVED** [review]
lgtm

**Pull request merged**